src/WordPress_Object/Clarkson_Role.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Clarkson Role.
+ */
+
+namespace Clarkson_Core\WordPress_Object;
+
+use Clarkson_Core\Objects;
+
+/**
+ * Object oriented wrapper for WP_Role objects.
+ */
+class Clarkson_Role {
+	/**
+	 * WordPress representation of this role object.
+	 *
+	 * @var \WP_Role
+	 */
+	protected $_role;
+
+	/**
+	 * Get Clarkson Role object by role name.
+	 *
+	 * @param string $name Role name.
+	 */
+	public static function get_by_name( $name ): ?Clarkson_Role {
+		$role = wp_roles()->get_role( $name );
+		if ( ! $role instanceof \WP_Role ) {
+			return null;
+		}
+
+		return Objects::get_instance()->get_role( $role );
+	}
```

Comment:
Kunnen we hier een `::get()` alias aan toevoegen. En een `get_many()` die `WP_Roles->role_objects` als array van Clarkson_Role teruggeeft?

Reply:
Ik ben het met je eens dat die functies er wel mogen komen. Ze gaan te vaak gebruikt worden denk ik.
Ookal zijn ze niet consistent, aangezien `::get()` nu ineens een string moet gaan accepteren, en `::get_many` geen array van arguments meer.